01_fib/mohitnag/main_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFibonacci(t *testing.T) {
+	assert := assert.New(t)
+	testCases := []struct {
+		number   int
+		expected []int
+	}{
+		{7, []int{1, 1, 2, 3, 5, 8, 13}},
```

Comment:
great work with the test cases.
Optional: if you want to be super perfectionist you could use a map and the key as a "sub test" name.
Here is a really good post explaining the details of that method:
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests